test/config.json

```diff
@@ -1,69 +1,25 @@
 {
 	"Tests": {
-		"check_make_parser": {
```

Comment:
based on offline discussion, I think we should add the two `check` tests back.

Reply:
Since those are added to github actions, I removed it from here. But I have to keep the `java` test here, as it needs docker dependency and this is possible via `go run test.go java` . Whereas these 2 checks/tests does not need any docker dependency.